### PR TITLE
Makes the Rapid Ligth Dispencer buildable

### PR DESCRIPTION
**Pull request by Baldir-Odensen**
## About The Pull Request
Makes the Rapid Light Dispenser buildable after researching its new prerequisite tech.

## Why It's Good For The Game
you will no longer have to spend 10 minutes just to light up a room that is missing light fixtures with lamps instead you can use the RLD to do it in under 1 minute leaving you to be able to do more interesting or pressing work like hull repairs or actually building stuff.

## Changelog
:cl:
add: The ability to build the RLD after researching its prerequisite tech.
/:cl:

### CODE REVIEW

File: code/modules/research/designs/tool_designs.dm [hunk 1]
```diff
@@ -63,6 +63,16 @@
 	category = list("Tool Designs")
 	departmental_flags = DEPARTMENTAL_FLAG_ENGINEERING
 
+/datum/design/rld
+	name = "Rapid Ligth Dispencer"
```
Comment (covertcorvid) [inline code/modules/research/designs/tool_designs.dm]: "Light" is misspelled here and in the description

File: code/modules/research/techweb/all_nodes.dm [hunk 2]
```diff
@@ -539,6 +539,15 @@
 	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 2500)
 	export_price = 5000
 
+/datum/techweb_node/rld
+	id = "rld"
+	display_name = "Advanced Sanitation Technology"
+	description = "Clean things better, faster, stronger, and harder!"
```
Comment (covertcorvid) [inline code/modules/research/techweb/all_nodes.dm]: Update these to be different from the janitor node

File: code/modules/research/techweb/all_nodes.dm [hunk 3]
```diff
@@ -539,6 +539,15 @@
...
+	prereq_ids = list("janitor")
+	design_ids = list("rld")
+	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 1000)
+	export_price = 5000
```
Comment (covertcorvid) [inline code/modules/research/techweb/all_nodes.dm]: Export price is typically 2x the cost, so 2000 here

File: code/modules/research/techweb/all_nodes.dm (hunk 2 above)
Comment (Baldir-Odensen) [reply code/modules/research/techweb/all_nodes.dm]: fuck forgot that

File: code/modules/research/techweb/all_nodes.dm (hunk 3 above)
Comment (Baldir-Odensen) [reply code/modules/research/techweb/all_nodes.dm]: ok fixing that

Comment (Baldir-Odensen) [reply code/modules/research/techweb/all_nodes.dm]: ok fixed that

File: code/modules/research/designs/tool_designs.dm (hunk 1 above)
Comment (Baldir-Odensen) [reply code/modules/research/designs/tool_designs.dm]: fixing em typos

File: code/modules/research/designs/tool_designs.dm [hunk 4]
```diff
@@ -63,6 +63,16 @@
 	category = list("Tool Designs")
 	departmental_flags = DEPARTMENTAL_FLAG_ENGINEERING
 
+/datum/design/rld
```
Comment (Karmic-Skink) [inline code/modules/research/designs/tool_designs.dm]: This should be in a separate file in the NSV13 folder.

File: code/modules/research/techweb/all_nodes.dm [hunk 5]
```diff
@@ -539,6 +539,15 @@
 	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 2500)
 	export_price = 5000
 
+/datum/techweb_node/rld
```
Comment (Karmic-Skink) [inline code/modules/research/techweb/all_nodes.dm]: This also should be in a separate file in the NSV13 folder.

File: code/modules/research/designs/tool_designs.dm (hunk 4 above)
Comment (Baldir-Odensen) [reply code/modules/research/designs/tool_designs.dm]: ok so i should add 2 files in the nsv13 folder 1 for tool designs and 1 for tech nodes?

Comment (Baldir-Odensen) [reply code/modules/research/designs/tool_designs.dm]: have done that now

File: nsv13/code/modules/rld_tech/rld_tech.dm [hunk 6]
```diff
@@ -0,0 +1,18 @@
+/datum/design/rld
+	name = "Rapid Light Dispencer"
```
Comment (Shadow-Quill) [inline nsv13/code/modules/rld_tech/rld_tech.dm]:
```suggestion
	name = "Rapid Light Dispenser"
```

File: nsv13/code/modules/rld_tech/rld_tech.dm [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+	desc = "Adds the Rapid Light Dispencer capable of constructing light fixtures and synthesising glowsticks."
```
Comment (Shadow-Quill) [inline nsv13/code/modules/rld_tech/rld_tech.dm]:
```suggestion
	desc = "Adds the Rapid Light Dispenser capable of constructing light fixtures and synthesising glow sticks."
```

**Comment by Vasily2013** [comment]
Is this PR approaching completion?

**Comment by Baldir-Odensen** [comment]
i think all of the typos are fixed so it should be complete.

**Pull request merged**